### ISSUE TITLE
[CS 1.6 - Feature Request] Add an option for old style fastswitch

### ISSUE THREAD

**Issue by konschta** [ValveSoftware/halflife#200]
Since the latest update hud_fastswitch behaves like its equivalents in Source and GO. In order to achieve a similar behaviour before you had to use something like the following workaround:

> bind "MWHEELDOWN" "invnext;+attack;wait;wait;-attack"
> bind "MWHEELUP" "invprev;+attack;wait;wait;-attack"

And though it may be more intuitive and beginner-friendly it provides less functionality.

Thus I propose the addition of an option for old style fastswitch, e.g. in the menu or through a new value for the "hud_fastswitch" command, like:
hud_fastswitch = **0** - no fastswitch
hud_fastswitch = **1** - new fastswitch behaviour
hud_fastswitch = **2** - old fastswitch behaviour

**tl;dr:**
- new update = less functionality
- add an option for old style fastswitch
